### PR TITLE
feat(Alerts): Updated the entire page and screenshots

### PR DESCRIPTION
**Pull request by nbaenam**
Updated the entire page and all the screenshots.
Added a new section, `Migrate your legacy Slack destinations to the new Slack destination`.

This is the ticket: [NR-101284](https://issues.newrelic.com/browse/NR-101284)

### CODE REVIEW

File: src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx [hunk 1]
```diff
@@ -514,145 +581,163 @@ Coming Soon: A new integration with New Relic Workflows will be available in the
...
 
-          * [Personal user tokens](https://support.pagerduty.com/docs/generating-api-keys#section-generating-a-personal-rest-api-key): If your account has advanced permissions, you can create unique personal REST API keys. Requests made using personal REST API keys are restricted to the user's permissions. If you choose to provide a user token API key, make sure it has the required permissions described above. [See PagerDuty instructions](https://support.pagerduty.com/docs/api-access-keys#generate-a-user-token-rest-api-key).
-    
-       <Callout variant="tip">
- For personal user tokens, we recommend you use a dedicated integration user that does not belong to a real user.
-    </Callout>
+  This integration requires a REST API Key. There are two types of REST API keys in PagerDuty:
+
+    * [General access key](https://support.pagerduty.com/docs/generating-api-keys#section-generating-a-general-access-rest-api-key): It includes all the above permission and can be obtained by PagerDuty Admins and Account Owners. [See PagerDuty instructions](https://support.pagerduty.com/docs/api-access-keys#generate-a-general-access-rest-api-key).
```
Comment (homelessbirds) [inline src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx:622]: Should this read:

"It includes all the permissions listed above"?

**Review by homelessbirds — APPROVED** [review]

File: src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx [hunk 2]
```diff
@@ -40,14 +42,23 @@ import accountsEmailConfWorkflow from 'images/accounts_screenshot-crop_email-con
...
 import accountsPagerDutyIntegrationOpened from 'images/accounts_screenshot-crop_pager-duty-integration-opened.webp'
 
 import accountsPagerDutyServiceDestination from 'images/accounts_screenshot-crop_pager-duty-service-destination.webp'
 
+import accountsMigrateSlackNewWorkflow from 'images/accounts_screenshot-crop_slack-new-workflow.webp'
+
+import accountsMigrateSlackRemoveLegacy from 'images/accounts_screenshot-crop_slack-remove-legacy.webp'
+
+
 Alerts and applied intelligence notification integrations are specific services and platforms you can use to send notifications from New Relic.
```
[the remark this answers is not in the document]
Comment (nbaenam) [reply src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx:62]: yes, you're right. I added your suggestion and this one as well: Through these notifications you get information about issues that need a review or that help you to take a decision about a detected problem.

File: src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx [hunk 3]
```diff
@@ -60,82 +71,96 @@ Read more about each of our specific notification integrations.
...
-    ### Two-way sync [#jira-two-way]
 
-    Two-way sync is applicable for workflows. To enable it, turn on the two-way integration toggle.
+ ### Two-way sync [#jira-two-way]
 
-    When turned on, a [Jira Webhook](https://developer.atlassian.com/server/jira/platform/webhooks/) is created in your Jira account at a later stage for the selected project. The webhook contains access details to New Relic (URL and API key).
+Two-way sync is applicable for workflows. To enable it, turn on the two-way integration toggle.
 
-    ### Sync with NewRelic Workflows [#two-way-sync-workflows]
+When turned on, a [Jira Webhook](https://developer.atlassian.com/server/jira/platform/webhooks/) is created in your Jira account at a later stage for the selected project. The webhook contains access details to New Relic (URL and API key).
```
[the remark this answers is not in the document]
Comment (nbaenam) [reply src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx:123]: Yes, you're right. I'm going to delete this part: "at a later stage for the selected project"

File: src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx [hunk 4]
```diff
@@ -145,8 +170,9 @@ Read more about each of our specific notification integrations.
   >
     
     <Callout variant="important">
-Coming Soon: A new integration with New Relic Workflows will be available in the ServiceNow app store. Click [here](https://forum.newrelic.com/s/hubtopic/aAX8W00000006D7/coming-soon-a-new-integration-between-new-relic-workflows-and-servicenow) to find out more.
+      From May 22, 2023, a new integration with New Relic Workflows is available in the ServiceNow app store. Click [here](https://forum.newrelic.com/s/hubtopic/aAX8W00000006D7/coming-soon-a-new-integration-between-new-relic-workflows-and-servicenow) to find out more.
```
[the remark this answers is not in the document]
Comment (nbaenam) [reply src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx:173]: Yes, I've removed the date and I've added "From now on, a new...."

**Pull request merged**